src/main/io/osd.c

```diff
@@ -153,6 +153,13 @@ static char osdGetAltitudeSymbol()
     }
 }
 
+static char osdGetBatterySymbol(int cellVoltage)
+{
+    /* Calculate current battery cell voltage as a percentage of range */
+    int voltPercent = ((cellVoltage - (batteryConfig()->vbatmincellvoltage * 10)) * 7) / ((batteryConfig()->vbatmaxcellvoltage - batteryConfig()->vbatmincellvoltage) * 10);
```

Comment:
Whis is easier to read:
```
scalerange(cellVoltage, batteryConfig()->vbatmincellvoltage * 10, batteryConfig()->vbatmaxcellvoltage * 10, 0, 7);
```
but rounding and mapping of edges may be improved (lowest symbol is used for voltage higher than `vbatmincellvoltage` due to rounding toward zero; `vbatmaxcellvoltage` is expected to be higher that fully charged cell, but rounding will work correctly here)